src/components/NavBar.js

```diff
@@ -0,0 +1,46 @@
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { FaChevronLeft, FaMicrophone } from 'react-icons/fa';
+import { RiSettings5Fill } from 'react-icons/ri';
+import './NavBar.css';
+
+const NavBar = ({ page, routeTo }) => {
+  <ul className="nav-header">
+    <li>
+      {
+        routeTo === 0
+          ? (
+            <>
+              <FaChevronLeft />
+              <span className="arrow-back">
+                2015
+              </span>
+            </>
+          ) : (
+            <NavLink to="/" className="back-link">
+              <FaChevronLeft />
+              <span className="arrow-back">
+                2015
+              </span>
+            </NavLink>
+          )
+      }
+    </li>
+    <li className="nav-title">
+      {page}
+    </li>
+    <li className="icons">
+      <FaMicrophone />
+    </li>
+    <li className="icons-last">
+      <RiSettings5Fill />
+    </li>
+  </ul>;
+};
```

Comment:
As requested by the previous reviewer, please make sure to add a `navbar` to your project 😉. Although you created the navbar component, it isn't being displayed on the page:
![image](https://user-images.githubusercontent.com/42799579/211071424-4b2864bb-b7cd-4fa5-bc5f-8154dbf5c279.png)

This is happening because your `NavBar` component isn't rendering anything:
![image](https://user-images.githubusercontent.com/42799579/211071490-ec0d176b-b054-4508-a255-374362990ff4.png)

To fix this issue, kindly add a render to your component 😄. You can do this in two easy ways:
1. Since your component doesn't contain any logic, only the desired render, you can change the bracket from `{ }` to `( )`, just like this:
![image](https://user-images.githubusercontent.com/42799579/211071760-ae19acc2-b952-4798-aabd-fe7b27c3bfa1.png)

2. You can also add the return statement:
![image](https://user-images.githubusercontent.com/42799579/211071961-58ed2cb3-9a60-44eb-868b-f344771ba607.png)

Both ways will result in your navbar being displayed:
![image](https://user-images.githubusercontent.com/42799579/211072046-5f3875c6-d6f5-4fd4-87fc-ca5d91b3df56.png)

Cheers and keep coding 😄!

Reply:
- [ ] This issue is well resolved :heavy_check_mark: